flexmeasures/data/models/planning/linear_optimization.py

```diff
@@ -270,8 +276,8 @@ def ems_flow_commitment_equalities(m, j):
         return (
             0,
             sum(m.commitment_quantity[:, j])
-            + sum(m.commitment_downwards_deviation[:, j])
-            + sum(m.commitment_upwards_deviation[:, j])
+            + sum(m.commitment_downwards_deviation[:, j, :])
```

Comment:
With the changes above, the indexes for `commitment_*_deviation` and `*_price` have changed from `(p,j)` to `(p,c,j)`, for this reason the **j** should be at the last position.